docs/src/misc.rst

```diff
@@ -523,6 +523,20 @@ API
 
     .. versionadded:: 1.8.0
 
+.. c:function:: int uv_os_environ(uv_env_item_t** envitems, int* count)
+
+    Retrieves all environment variables. This function will allocate memory
+    which must be freed by calling :c:func:`uv_os_free_environ`.
+
+    .. versionadded:: 1.31.0
```

Comment:
Can you add a `This function is not thread safe` warning? You can copy/paste it from uv_os_getenv() below.